chapters/1.md

```diff
@@ -11,33 +11,39 @@ Maria is purchasing a car from George, and in this chapter we will consider thre
 
-It is easy to be confused about the terminology at first, especially since most people are simultaneously introduced to several jargony concepts. You can think about "blockchains" as a technology that allows networks to establish "decentralized consensus" agreements. By enabling strangers to safely share a ledger, it becomes possible to build "cryptocurrencies" that function as digital cash. There are a multitude of regular currencies (euros, dollars, yen, etc); analogously, various teams have built many different cryptocurrencies (Monero, Ethereum, Bitcoin, etc).
+One significant obstacle in creating such a system was that there was no known way to prevent people from spending the same money twice without having a central party to validate each transaction.  Given that some network participants may try to lie, cheat, pretend to be multiple people, etc, how do you maintain a single version of the ledger to prevent double spending?  In computing, this is illustrated in a thought experiment referred to as the “Byzantine Generals problem” meant to demonstrate the pitfalls and design challenges of attempting to coordinate an action by communicating over an unreliable network. In the experiment, two generals are only able to communicate with one another by sending a messenger through enemy territory. The experiment asks how they might reach an agreement on the time to launch an attack, while knowing that any messenger they send could be captured.
+
+Thankfully, in 2009 with the launch of Bitcoin, Satoshi Nakamoto (a pseudonym) demonstrated a suitable combination of technologies, game theory and economics which mitigated the above issues.  This gave the world an open, verifiable, trustless currency which anyone could use, but no single entity controlled.  In the first 4 years of its existence, a number of shortcomings were identified in Bitcoin's original design and implementation. In 2013 Nicolas van Saberhagen (also a pseudonym) published the CryptoNote 2.0 whitepaper with the goal of addressing these shortcomings and provide a superior digital cash payment system.  Six months later, Monero was launched as an implementation of the CryptoNote protocol.
+
+#### Cryptocurrency in a nutshell
+
+When learning about cryptocurrencies for the first time, most people will feel overwhelmed by the complex jargon and new concepts. As a starter, you can think about "blockchains" as a technology which allows networks to establish "decentralized consensus" (agreement about who owns what). By enabling strangers to safely share a ledger, it becomes possible to build "cryptocurrencies" that function as digital cash. Just as there are a multitude of government currencies (euros, dollars, yen, etc), there are also many different cryptocurrencies (Monero, Ethereum, Bitcoin, etc).
```

Comment:
The definitions given by these paragraphs are really general and do not guide the reader into understanding it. I would keep the text from first edition.

Reply:
I have reverted these, but have a couple concerns.

I added the header as it seemed like an abrupt change which did not fit with the flow of the text, almost like this paragraph should be an aside (a note to the reader).  Maybe there is a better way to integrate it?
My first draft prefixed the paragraph with "Note to reader:" for example.

Also, I was concerned that **_"decentralized consensus" agreements_** also feels quite jargony itself, that was why I explained it as **_"decentralized consensus"  (agreement about who owns what)_**.  I can't imaging people new to this subject understanding what "decentralized consensus" agreements are.